WORKING_GROUPS.md

```diff
@@ -222,14 +220,14 @@ By making a contribution to this project, I certify that:
 
 ### Moderation Policy
 
-The [ASC Software Organization Moderation Policy] applies to this WG.
+The [Planetary Software Organization Moderation Policy] applies to this WG.
 
 ### Code of Conduct
 
-The [ASC Software Organization][] applies to this WG.
+The [Planetary Software Organization][] applies to this WG.
```

Comment:
Does it make sense to put the org link in now that this is stabler?

Reply:
Yes.

Reply:
Sorry, I was leaving on a vacation: Yes, but the 'code of conduct' needs to be sorted out for 'Planetary Software' and right now that's not set.